src/System.Management.Automation/engine/Types_Ps1Xml.cs

```diff
@@ -236,6 +236,8 @@ public static IEnumerable<TypeData> Get()
             td24.Members.Add("IPV6Address",
                 new ScriptPropertyData(@"IPV6Address", GetScriptBlock(@"$iphost = [System.Net.Dns]::GetHostEntry($this.address)
           $iphost.AddressList | Where-Object { $_.AddressFamily -eq [System.Net.Sockets.AddressFamily]::InterNetworkV6 } | Select-Object -first 1"), null));
+            td24.Members.Add("Destination",
```

Comment:
Since `Test-Connection` will be rewritten, do we still need this change?

Reply:
I don't use Win32_PingStatus - we can remove this here.

Reply:
removed